sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -196,24 +196,25 @@ def default_label(self):
 
     def expand(self, pcoll):
       compare = self._compare
-      if (not self._args and not self._kwargs and
-          not self._key and pcoll.windowing.is_default()):
+      if (not self._args and not self._kwargs and 
```

Comment:
Nit: This fits in one line, or?

Reply:
It was failing the lint with 1 extra character.